src/Components/Components/src/Properties/ILLink.Substitutions.xml

```diff
@@ -2,18 +2,12 @@
   <assembly fullname="Microsoft.AspNetCore.Components" >
     <!-- HotReload will not be available in a trimmed app. We'll attempt to aggressively remove all references to it. -->
     <type fullname="Microsoft.AspNetCore.Components.RenderTree.Renderer">
+      <!-- Renderer.IsHotReloading will always return false in a trimmed app. -->
+      <method signature="System.Boolean get_IsHotReloading()" body="stub" value="false" />
```

Comment:
Is it also possible to short-circuit `RenderHandle.IsHotReloading` to be always-false so that, in production, we can skip both the null check and the property getter call that it does on each call to `SetParametersAsync`?

I know this is small beans but it would be super nice to have zero overhead for this feature in production.